server/index.js

```diff
@@ -1,39 +1,26 @@
-const express = require('express');
-const path = require('path');
-const morgan = require('morgan');
-const fetch = require('node-fetch');
+var express = require('express');
```

Comment:
For required modules, you don't want them to be mutable, hence it's best to declare these with const. I.e const express = require("express"), const path = require("path").

**Rationale:** To prevent possible bugs or conflicts in your code (i.e accidentally assigning another variable with the name express), it would be wise to set these variables as unchangeable.

All it takes is to change vars to const or let depending on your use case.